content/docs/release-notes.md

```diff
@@ -2,7 +2,60 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: Back up WAL to S3 for disaster recovery.
+
+* Safekeeper: Download WAL from S3 on demand.
+
+* Propagate postgres authentication errors to the user.
+
+* Rename `contrib/zenith` to `contrib/neon`.
```

Comment:
```suggestion
* Postgres: Rename `contrib/zenith` to `contrib/neon`.
```

Reply:
BTW, do we really need it in the release notes? looks like completely internal thing